app/experimenter/experiments/models/nimbus.py

```diff
@@ -242,10 +249,10 @@ def delete_branches(self):
 
     @property
     def should_allocate_bucket_range(self):
-        return self.status in [
-            NimbusExperiment.Status.PREVIEW,
-            NimbusExperiment.Status.REVIEW,
-        ]
+        return (
+            self.status == NimbusExperiment.Status.PREVIEW
+            and self.publish_status == NimbusExperiment.PublishStatus.APPROVED
```

Comment:
I think `or` here, we want to allocate if it's going to preview or if it's being launched.